src/_tests_/components/forecast-summary.test.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { cleanup, render } from "@testing-library/react";
+import ForecastSummary from "../../components/forecast-summary";
+
+afterEach(cleanup);
+
+describe("ForecastSummary", () => {
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <ForecastSummary
+      date="1525046400000" 
+      temperature={11}
+      description="mockDescription"
+      icon="800"
+      />
+    );
+
+    expect(asFragment).toMatchSnapshot();
+  });
+
+  it("renders the correct props", () => {
+    const { getByTestId } = render(
+      <ForecastSummary
+      date="1525046400000"
+      temperature={13}
+      description="mockDescription"
+      icon="200"
+      />
+    );
+    expect(getByTestId("date-id")).toHaveClass("date");
```

Comment:
I would test the prop appears correctly for a test checking the correct props are rendered.